components/prism-abap.js

```diff
@@ -0,0 +1,50 @@
+		lookbehind: true,
+	},
+	/* Numbers can be only integers. Decimal or Hex appear only as strings */
+	'number' : /\b\d+\b/g,
+	/* Operators must always be surounded by whitespace, they cannot be put 
+	adjacent to operands. 
+	*/
+	'operator' : {
+		pattern: /(\s)(\+|-|\*{1,2}|\/|<|>|<=|>=|=|\?=|<>)(?=\s)/g,
+		lookbehind: true,
+	},
+	'string_operator' : {
+		pattern: /(\s)(&{1,2})(?=\s)/g,
+		lookbehind: true,
+		alias: "keyword"
+	},
+	'token_operator' : [{
+		/* Special operators used to access structure components, class methods/atributes, etc...*/
+		pattern: /(\w)(-|~|->|=>|\||\{|\})(?=\w)/g,
+		lookbehind: true,
+		alias: "punctuation"
+	}, {
+	    /*Special tokens used do delimit string templates*/
+	    pattern: /[\|\{\}]/g,
```

Comment:
There is no need to escape those characters inside a character class. The regexp can be rewritten as `/[|{}]/`.